repair_tests/incremental_repair_test.py

```diff
@@ -207,6 +208,7 @@ def test_manual_session_fail(self):
         self.fixture_dtest_setup.setup_overrides.cluster_options = ImmutableMapping({'hinted_handoff_enabled': 'false',
                                                                                      'num_tokens': 1,
                                                                                      'commitlog_sync_period_in_ms': 500})
+        self.fixture_dtest_setup.init_default_config()
```

Comment:
> I believe `self.init_default_config()` and `self.fixture_dtest_setup.init_default_config()` are synonymous

Hmm yeah, so it is. I find that a little non-obvious and easy to miss, but ¯\_(ツ)_/¯

> Is it because the config changes in the preceding lines weren't actually being picked up prior?

Exactly, the fixture initialization order means that the cluster config has already been set by this point and so none of these options were being used in the existing tests.

Reply:
Yea the proxying to the fixture is a bit non-intuitive. iirc it was added to make the transition from nosetest -> pytest easier.